packages/prisma/src/prisma/schema/participant.prisma

```diff
@@ -38,7 +38,7 @@ model ParticipantAccount {
   participantId String      @db.Uuid
 
   createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  updatedAt DateTime @default(now()) @updatedAt
```

Comment:
Does this change not cause the prisma schema to be out of date with the actual postgres database due to different defaults, leading to potential crashes?

Reply:
Additionally, I am relatively sure, that this already happens automatically through the @updatedAt tag - we use this nowhere else

Reply:
The migration on staging failed because the field was added as required but is empty in the existing rows. Hence the addition. Could also be done with a script but not sure what we gain by that